doc/source/indexing.rst

```diff
@@ -481,12 +487,52 @@ A list of indexers where any element is out of bounds will raise an
 
 .. code-block:: python
 
-   dfl.iloc[[4,5,6]]
+   dfl.iloc[[4, 5, 6]]
    IndexError: positional indexers are out-of-bounds
 
-   dfl.iloc[:,4]
+   dfl.iloc[:, 4]
    IndexError: single positional indexer is out-of-bounds
 
+.. _indexing.callable:
+
+Selection By Callable
+---------------------
+
+.. versionadded::0.18.1
+
+``.loc``, ``.iloc``, ``.ix`` and also ``[]`` indexing can accept a ``callable`` as indexer. The ``callable`` must
+be a function with one argument which accepts caller and returns valid output for indexing.
```

Comment:
Could you change "one argument  which accepts caller" to "one argument, the calling DataFrame or Series"?